### ISSUE TITLE
Affinity Loss usage

### ISSUE THREAD

**Issue by ChidanandKumarKS** [yoshitomo-matsubara/torchdistill#181]
Hi @yoshitomo-matsubara 
I want to do semantic segmentation using distillation using "AffinityLoss" as in the paper "Knowledge Adaptation for Efficient Semantic Segmentation".

Its been shown in the line
 [class AffinityLoss(nn.Module):](https://github.com/yoshitomo-matsubara/torchdistill/blob/5377be466c9460e0125892aa0d92aeb86418c752/torchdistill/losses/single.py#L892)

I tried to use example section mentioned in 
python3 examples/semantic_segmentation.py --config configs/sample/coco2017/multi_stage/ktaad/lraspp_mobilenet_v3_large_from_deeplabv3_resnet50.yaml --log log/coco2017/ktaad/lraspp_mobilenet_v3_large_from_deeplabv3_resnet50.txt
 
But while debugging it will never excute AffinityLoss module.
requesting you to help us how to go ahed. Seeking your help

**Comment by yoshitomo-matsubara** [comment]
Hi @ChidanandKumarKS 

How did you confirm that it will never execute AffinityLoss module? If you change any part of code or yaml file, please share them here.

Note that the yaml config is for a multi-stage training, and [AffiniyLoss will be used only in the 2nd stage](https://github.com/yoshitomo-matsubara/torchdistill/blob/main/configs/sample/coco2017/multi_stage/ktaad/lraspp_mobilenet_v3_large_from_deeplabv3_resnet50.yaml#L154-L253) after [the 1st stage training (1 epoch)](https://github.com/yoshitomo-matsubara/torchdistill/blob/main/configs/sample/coco2017/multi_stage/ktaad/lraspp_mobilenet_v3_large_from_deeplabv3_resnet50.yaml#L67-L153).

**Comment by ChidanandKumarKS** [comment]
@yoshitomo-matsubara 
Sorry the training was quite long time. As you mentioned it is 2-stage training, iam able to debug for the section of the code based on Affinity loss. Its working.

Thanks
Kumarr
